.github/workflows/cloudevents-integration.yml

```diff
@@ -0,0 +1,32 @@
+name: CloudEventsIntegration
+
+on:
+  workflow_dispatch: {}
+  pull_request:
+    paths:
+     - 'pkg/addonmanager/cloudevents/*.go'
+     - 'test/integration/cloudevents/**'
+    branches:
+      - main
+      - release-*
+
+env:
+  GO_VERSION: '1.21'
+  GO_REQUIRED_MIN_VERSION: ''
+
+permissions:
+  contents: read
+
+jobs:
+  integration:
```

Comment:
can we also add an e2e to at least check that the example cloud event addons are available when deploying into a kubernetes cluster?

Reply:
e2e test was removed in this commit: https://github.com/open-cluster-management-io/addon-framework/pull/256/commits/d8d4f5269b42f4e24b4917ce38d372da62aa56a6
because the work client based on cloudevents drivers (sdk-go) are missing GC machnism, we will revisit and add e2e after GC machnism support for cloudevents drivers.